docs/Homebrew-and-Python.md

```diff
@@ -9,7 +9,12 @@ Homebrew provides formulae to brew a more up-to-date Python 2.7.x and 3.x.
 **Important:** If you choose to install a Python which isn't either of these two (system Python or brewed Python), the Homebrew team can only provide limited support.
 
 ## Python 2.x or Python 3.x
-Homebrew provides a formula for Python 2.7.x and one for Python 3.x. They don't conflict, so they can both be installed. The executable `python2` will always point to the 2.x and `python3` to the 3.x version. The executable `python` will always point to the macOS system Python.
+Homebrew provides one formula for Python 2.7.x and another for Python 3.x. The executables are organized as follows so that Python 2 and Python 3 can both be installed without conflict:
+* `python` will always point to the macOS system Python
```

Comment:
I appreciate the wording exists already but "always" seems misleading here given:
```
This formula installs a python2 executable to /usr/local/bin.
If you wish to have this formula's python executable in your PATH then add
the following to ~/.zshrc:
  export PATH="/usr/local/opt/python/libexec/bin:$PATH"
```
Perhaps, "by default" instead.